### DISCUSSION TITLE
Tasmota pair with RM433

### DISCUSSION THREAD

**Discussion by xeac** [arendst/Tasmota#13231] [Q&A] [answered]
hi,

Can someone tell me if i can pair tasmotized Sonoff 4ch Pro R2 and Sonoff RM433, without using a Rf bridge? 
I couldn't find helpfull info.

thanks,

**Comment by sfromis** [comment, marked as answer]
That is actually not a Tasmota question, as the pairing is done outside the control of the ESP8266. Thus having Tasmota flashed should just not make any difference for whether you can pair it or not. I'd expect the pairing to work, but ... Murphy!
